src/core/components/operation-tag.jsx

```diff
@@ -56,8 +57,7 @@ export default class OperationTag extends React.Component {
     let showTag = layoutSelectors.isShown(isShownKey, docExpansion === "full" || docExpansion === "list")
 
     return (
-      <div className={showTag ? "opblock-tag-section is-open" : "opblock-tag-section"} >
-
+      <div className={`opblock-tag-section ${showTag ? "is-open" : ""}`}>
```

Comment:
**Change request**: use `classnames` instead of ternary operators here, example: https://github.com/swagger-api/swagger-ui/blob/ad786b023f8e54fcdb4a883039425148d8a8a327/src/core/json-schema-components.jsx#L258

it does have a lot of other usage patterns, see [here](https://github.com/JedWatson/classnames#usage) for documentation!